tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
<p>I am not sure I am seeing the notebook correctly in this viewer, but if so, this is partly an exact repetition of a paragraph below. Copy&amp; Paste lapse?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Yep that's a copy-paste mistake, fixed now</p><p> </p>